src/plugins/intel_gpu/src/kernel_selector/kernels/sdpa/sdpa_kernel_micro.cpp

```diff
@@ -254,7 +254,7 @@ void SDPAKernelMicro::init_microkernels(const sdpa_params& params, micro::Packag
     const auto& V = params.inputs[2];
 
     auto& out = params.outputs[0];
-    const auto head_size = params.conf.head_size;
+    const auto head_size = params.conf.k_head_size;
```

Comment:
nit: proposed to rename head_size to k_head_size to improve readability (k_head... and v_head...  instead of head and v_head pair)